lib/gui/modules/analytics.js

```diff
@@ -39,6 +40,22 @@ const analytics = angular.module(MODULE_NAME, [
   require('../models/settings')
 ]);
 
+/**
+ * @summary Get OS architecture
```

Comment:
Maybe the summary should make it clear that we're talking about the host OS architecture?